tests/model_integration_tests.py

```diff
@@ -102,13 +110,213 @@ def test_dummy_cache(self):
             'django_cache_manager.cache_backend': {
                 'BACKEND': 'django.core.cache.backends.dummy.DummyCache',
             }
-        }       
+        }
         with override_settings(DEBUG=True, CACHES=CACHES):
             for i in range(5):
-                len(Manufacturer.cached_objects.all())
-            self.assertEquals(5, len(connection.queries))
+                len(Manufacturer.objects.all())
+            self.assertEqual(len(connection.queries), 5)
+
+    @override_settings(DEBUG=True)
```

Comment:
Can group one-to-one tests in a one to one test case. Same for many-to-one and many-to-many